### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by amartya-rs**
updated

### CODE REVIEW

**Review by Sarbajaya — COMMENTED** [review]
great work

**Comment by amartya-rs** [comment]
> great work

thanks

**Comment by amartya-rs** [comment]
thanks

**Review by Sarbajaya — APPROVED** [review]
great work

**Comment by amartya-rs** [comment]
> great work

thanks for your feedback

**Review by Sarbajaya — CHANGES_REQUESTED** [review]
use proper spacing

**Comment by amartya-rs** [comment]
> use proper spacing

cool

File: index.html [hunk 1]
```diff
@@ -10,5 +10,7 @@
     <div>dev branch testing</div>
     <p>some change</p>
     <div>Avatar</div>
+
+    <p>this a secondary branch</p>
```
Comment (Sarbajaya) [inline index.html:14]: better semantic html

**Review by Sarbajaya — APPROVED** [review]

**Pull request closed without merging**